### PR TITLE
created hp_paragraph update script

### PR DESCRIPTION
**Pull request by ben2collins**

### CODE REVIEW

File: drupal/acquia/update_hp_paragraphs.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#Run composer update on all local codebases. 
+composer update
```
Comment (drogers98) [inline drupal/acquia/update_hp_paragraphs.sh]: If you run composer update here, you do not need to run it again at howard/howard_paragraphs

Comment (drogers98) [reply drupal/acquia/update_hp_paragraphs.sh]: ie, running composer update, without a specific package, updates "everything", including howard_paragraphs

**Comment by drogers98** [comment]
My recommendation would be to dump the general "composer update" since that updates all core/contrib/everything, and specifically focus on adding the howard modules.

**Comment by drogers98** [comment]
Ben, I'm not sure the usefulness of a script that only updates 1 module. Are you going to include "all howard repos"? ie, all those listed here: https://packagist.org/packages/howard/

**Comment by ben2collins** [comment]
When I originally wrote this script I was following the instructions (Item 1 in particular)
from Bugify ticket #22972. 

Screen Shot 2019-12-09 at 12 10 24 PM

I agree and I'll add updates for the other Howard packages.

**Comment by ben2collins** [comment]
Also Antonio recommended I keep my branch open on the original repo so we can track all changes under one roof. So I created a new pull request but this time it doesn't come from a forked repo (https://github.com/howard-university-web-services/Howard-Automation-Library/pull/3)

**Pull request closed without merging**